Source/WebKit/UIProcess/API/glib/WebKitUserContentManager.cpp

```diff
@@ -96,6 +96,8 @@ static void webkit_user_content_manager_class_init(WebKitUserContentManagerClass
      * webkit_user_content_manager_register_script_message_handler()
      *
      * Since: 2.8
+     *
+     * Deprecated: 2.40
```

Comment:
I think it would be nice to add a small note to the deprecated functions/signals, indicating what the new replacement is, something like:

```suggestion
     *
     * Deprecated: 2.40
     *
     * New code should use the [signal@UserContentManager::script-message-with-reply-received] instead.

```